src/components/shared/SvgSprite.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+
+interface TProps {
+  id: string;
+  size?: [number | string, number | string];
+  color?: string;
+}
+
+const SvgSprite: FC<TProps> = props => {
```

Comment:
```suggestion
const SvgSprite: FC<TProps> = ({ id = '', size = ['auto', 'auto'], color = 'white' }) => {
```

and why id = '' by default

Reply:
![image](https://github.com/Egorkostuck/test-task-frontandiya-/assets/69008147/50986bb8-6a54-4532-8183-2e58c6f2627d)
Because I use SvgSprite for optimization. In this component you need to specify the id of the svg you want to return